app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -1071,16 +1071,17 @@ private void populateResultsBox(IVScanResult ivScanResult) {
 
     /**
      * Adjusts expandedLevelSeekbar and expandedLevelSeekbar thumbs
-     * <p/>
+     *
      * expandedLevelSeekbar is a single thumb seekbar
      * Seekbar should be max at possible Pokemon level at trainer level 40.
      * Thumb should be placed at current Pokemon level
-     * <p/>
+     *
      * expandedLevelSeekbarBackground is a double thumb seekbar
      * Seekbar should be max at possible Pokemon at trainer level 40
      * Thumb 1 should be marked as an orange marker and placed at the max possible Pokemon level at the current
      * trainer level
      * Thumb 2 should be invisible and placed at the max
+     * Set empty on touch listener to prevent changing values of Thumb 1
```

Comment:
Maybe this should probably be an implementation comment, not part of the JavaDoc. I know the same is true for almost this whole JavaDoc, so I guess I'm not gonna insist.